marklogic-data-hub-central/src/test/resources/test-projects/reference-project/entities/Customer.entity.json

```diff
@@ -10,6 +10,9 @@
         "name"
       ],
       "properties": {
+        "customerId": {
```

Comment:
Just an FYI, we have the reference project in both core and central - I'll get that resolved soon, as central can definitely reuse the one in core.

Reply:
Updated the hub central code and tests to use the reference model project from the core.

I haven't deleted the project since there's a test that does use the flows and other configs from the HC's reference project and I didn't want to copy all of that baggage over to the reference model project in the core.